### ISSUE TITLE
I have the hardware and the balls

### ISSUE THREAD

**Issue by unmanagedtn** [batchmcnulty/Malbait#1]
I have an dell r720 with 16 cores and 256gb ram if thats bulky enough? And I can put it out on a 250 mbit connction for testing ...

should I give it a go ?

**Comment by batchmcnulty** [comment]
Yeah go for it man. Sorry I didn't notice your post earlier - been busy on other projects.
I'm just about to roll in an update that will email you when it trips (optionally) for maximum potential self-ownage! Wanna be my guinea pig? (running off a fone so I can't use sendmail, also I don't want to)
